lua/astrocommunity/media/image-nvim/README.md

```diff
@@ -0,0 +1,22 @@
+# image.nvim
+
+This plugin attempts to add image support to Neovim.
+
+**Repository:** <https://github.com/3rd/image.nvim>
+
+Works with Kitty + Tmux, and it handles painful things like rendering an image at a given position in a buffer, scrolling, windows, etc.
+
+It has built-in Markdown and Neorg integrations that you can use right now.
+It can also render image files as images when opened.
+
+**Requirements**
+
+These are things you have to setup on your own
+
+* ImageMagick - **mandatory**
+* magick LuaRock - **mandatory** (luarocks --local --lua-version=5.1 install magick)
+* Kitty >= 28.0 - for the kitty backend
+* ueberzugpp - for the ueberzug backend
+* curl - for remote images
+
+**This plugin is configured to be used with kitty terminal and lua 5.1**
```

Comment:
I don't think lua 5.1 is mentioned explicitly, so that should  be dropped.

Reply:
At the moment lua 5.1 is in a way directly referenced, since the first two lines in the init.lua are referencing like so:

```
package.path = package.path .. ";" .. vim.fn.expand "$HOME" .. "/.luarocks/share/lua/5.1/?/init.lua;"
package.path = package.path .. ";" .. vim.fn.expand "$HOME" .. "/.luarocks/share/lua/5.1/?.lua;"
```

This is exactly the same as the plugin does it.
Not sure if it works with a higher version of lua ( might be ) but untested.

These lines can also be somewhere else, ( polish function for instance? ) but then it becomes even more manual configuration.

Maybe there is a more "astro" way to do this?

Reply:
> At the moment lua 5.1 is in a way directly referenced, since the first two lines in the init.lua are referencing like so:
> 
> ```
> package.path = package.path .. ";" .. vim.fn.expand "$HOME" .. "/.luarocks/share/lua/5.1/?/init.lua;"
> package.path = package.path .. ";" .. vim.fn.expand "$HOME" .. "/.luarocks/share/lua/5.1/?.lua;"
> ```
> 
> This is exactly the same as the plugin does it. Not sure if it works with a higher version of lua ( might be ) but untested.
> 
> These lines can also be somewhere else, ( polish function for instance? ) but then it becomes even more manual configuration.
> 
> Maybe there is a more "astro" way to do this?

We should do this in a better way. I’ll have to think on it a bit.

Reply:
Ok good.
yeah, probably not the best way to hardcode this into the plugin...

Reply:
Ah seems to be that the limitation of lua 5.1 is imposed by the luarock bit. magick's lua bindings actually requires luajit